resources/views/user/index.blade.php

```diff
@@ -1,48 +1,12 @@
 <x-app-layout>
 
 {{-- View Added for All Chirpers --}}
-<div class="mt-6 bg-white shadow-sm rounded-lg divide-y" style="background-color:aliceblue;">
-                <div class="p-6 flex space-x-2">
-                    <div class="flex-1">
-
-                        <div class="flex justify-between items-center">
-                            <div>
-                                <h2>Profile Image</h2>
-                            </div>
-                            <div>
-                                <h2>Name</h2>
-                            </div>
-                            <div>
-                                <h2>Account Creation Date</h2>
-                            </div>
-                            <div>
-                                Total Post Count
-                            </div>
-                        </div>
-
-                    </div>
-                </div>
-
+<div class="mt-6 bg-white shadow-sm rounded-lg" style="background-color:aliceblue; width:50%;display:flex; justify-content:center;">
```

Comment:
If your background color and max width are going to be site wide move this css to the layout